cdap/http.go

```diff
@@ -37,13 +38,26 @@ func urlJoin(base string, paths ...string) string {
 	return fmt.Sprintf("%s/%s", strings.TrimRight(base, "/"), strings.TrimLeft(p, "/"))
 }
 
-func httpCall(client *http.Client, req *http.Request) ([]byte, error) {
-	log.Printf("%+v", req)
-
+func do(client *http.Client, req *http.Request) (*http.Response, error) {
```

Comment:
nit: doHttpCall and move this below (the code in this provider reads top down like english).